src/content/configuration/experiments.mdx

```diff
@@ -184,6 +184,28 @@ Detect changes to remote resources and upgrade them automatically.
 
 Enable native CSS support. Note that it's an experimental feature still under development and will be enabled by default in webpack v6, however you can track the progress on [GitHub](https://github.com/webpack/webpack/issues/14893).
 
+- Type:
+
+  - `boolean`
+  - `CssExperimentOptions`
+
+    ```ts
+    {
+      exportsOnly?: boolean;
+    }
+    ```
+
+Experimental features:
+
+- Inbuilt support for CSS and CSS Modules. Use the `.module.css` extension for CSS Modules.
```

Comment:
We need to clarify what you shoudl set `css/auto` to enable this, I think in future we should use `css/auto` by default for any CSS files

Reply:
based on the filename, we automatically use css modules rule
https://github.com/webpack/webpack/blob/dc4d97429b2013f7770ba785721b65cf82c2ef04/lib/config/defaults.js#L728-L729
dont need to set css/auto explicitly

Reply:
Oh, yeah, missing that